plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHive2PartitionsOnDataLake.java

```diff
@@ -15,10 +15,10 @@
 
 import io.trino.plugin.hive.containers.HiveHadoop;
 
-public class TestHive2InsertOverwrite
-        extends BaseTestHiveInsertOverwrite
+public class TestHive2PartitionsOnDataLake
```

Comment:
mind "Partitions" when renaming the base class

Reply:
Let's continue discussion in other comment thread https://github.com/trinodb/trino/pull/10385#discussion_r783826360